starter-core/src/main/java/io/micronaut/starter/feature/aws/template/cdkappstack.rocker.raw

```diff
@@ -140,6 +142,11 @@ public class AppStack extends Stack {
                 .memorySize(512)
                 .tracing(Tracing.ACTIVE)
                 .logRetention(RetentionDays.ONE_WEEK)
+@if (architecture == Architecture.ARM) {
+                .architecture(Architecture.ARM_64)
+} else {
```

Comment:
See https://github.com/micronaut-projects/micronaut-starter/blob/3.7.x/CONTRIBUTING.md#avoid-empty-else and avoid this empty else.

Reply:
Resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1466/commits/49da2f8f0ea614d25759c48846db10cb46d2a88a